mito-ai/src/Extensions/SettingsManager/database/model.ts

```diff
@@ -28,12 +28,21 @@ export interface DatabaseConfig {
     fields: DatabaseField[];
 }
 
+const ALIAS_PLACEHOLDER = 'Enter a nickname for this database';
+
 export const databaseConfigs: Record<string, DatabaseConfig> = {
     mssql: {
         type: 'mssql',
         displayName: 'Microsoft SQL Server',
         alertText: 'Microsoft SQL Server requires an additional driver. If you\'ve already installed it, you can safely ignore this message. For more info, consult the <a href="https://docs.trymito.io/mito-ai/database-connectors/microsoft-sql-server" target="_blank">Mito docs</a>.',
         fields: [
+            {
+                name: 'alias',
```

Comment:
I think alias is maybe too technical? What if we just call this `nickname` to make it clear that this is just any name they want to give it as opposed to some name they need to go find.